met-cron/migrations/versions/fe0a86c0d458_update_value_columns_size.py

```diff
@@ -0,0 +1,67 @@
+"""Update value columns size
+
+Revision ID: fe0a86c0d458
+Revises: c03b11293c59
+Create Date: 2022-08-29 14:09:01.479279
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'fe0a86c0d458'
+down_revision = 'c03b11293c59'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column('user_response_detail', 'engagement_id',
+               existing_type=sa.INTEGER(),
+               nullable=True)
+
```

Comment:
do you know whats this one?

Reply:
The billable warning I think, it was auto generated

Reply:
Nullable